Extraneous modules when npm shrinkwrapped

When I install v2.1.0 in a downstream, I get the following error upon an `npm ls`

```
npm ERR! extraneous: @bahmutov/parse-github-repo-url@0.1.2 /Users/cfelder/livingdocs/livingdocs-server/node_modules/@livingdocs/docker/node_modules/@bahmutov/parse-github-repo-url
npm ERR! extraneous: @semantic-release/commit-analyzer@2.0.0 /Users/cfelder/livingdocs/livingdocs-server/node_modules/@livingdocs/docker/node_modules/@semantic-release/commit-analyzer
npm ERR! extraneous: @semantic-release/last-release-npm@1.2.1 /Users/cfelder/livingdocs/livingdocs-server/node_modules/@livingdocs/docker/node_modules/@semantic-release/last-release-npm
npm ERR! extraneous: @semantic-release/release-notes-generator@2.0.0 /Users/cfelder/livingdocs/livingdocs-server/node_modules/@livingdocs/docker/node_modules/@semantic-release/release-notes-generator
npm ERR! extraneous: git-head@1.14.0 /Users/cfelder/livingdocs/livingdocs-server/node_modules/@livingdocs/docker/node_modules/git-head
npm ERR! extraneous: github@0.2.4 /Users/cfelder/livingdocs/livingdocs-server/node_modules/@livingdocs/docker/node_modules/github
npm ERR! extraneous: hide-secrets@1.1.0 /Users/cfelder/livingdocs/livingdocs-server/node_modules/@livingdocs/docker/node_modules/hide-secrets
npm ERR! extraneous: lodash@4.7.0 /Users/cfelder/livingdocs/livingdocs-server/node_modules/@livingdocs/docker/node_modules/lodash
npm ERR! extraneous: nerf-dart@1.0.0 /Users/cfelder/livingdocs/livingdocs-server/node_modules/@livingdocs/docker/node_modules/nerf-dart
npm ERR! extraneous: node-emoji@1.2.1 /Users/cfelder/livingdocs/livingdocs-server/node_modules/@livingdocs/docker/node_modules/node-emoji
npm ERR! extraneous: npmconf@2.1.2 /Users/cfelder/livingdocs/livingdocs-server/node_modules/@livingdocs/docker/node_modules/npmconf
npm ERR! extraneous: require-relative@0.8.7 /Users/cfelder/livingdocs/livingdocs-server/node_modules/@livingdocs/docker/node_modules/require-relative
npm ERR! extraneous: run-auto@1.1.3 /Users/cfelder/livingdocs/livingdocs-server/node_modules/@livingdocs/docker/node_modules/run-auto
npm ERR! extraneous: run-series@1.1.4 /Users/cfelder/livingdocs/livingdocs-server/node_modules/@livingdocs/docker/node_modules/run-series
npm ERR! extraneous: travis-after-all@1.4.4 /Users/cfelder/livingdocs/livingdocs-server/node_modules/@livingdocs/docker/node_modules/travis-after-all
```

The problem only occurs when the published package is shrinkwrapped. I manually shrinkwrapped and published (equals the process for 2.1.0 the release was done automatically via semantic-release) 2.1.1, which yields the same error. The manually published 2.1.2 *without shrinkwrapping" works like expected.

--- a/bin/semantic-release.sh
+++ b/bin/semantic-release.sh
@@ -4,6 +4,9 @@ set -e
 
 semantic-release pre
 
+# Work around shrinkwrapping issues
+npm prune --production
+
 # Release new version
 npm shrinkwrap
 npm publish

